Level-1/axpy/axpy.cpp

```diff
@@ -0,0 +1,154 @@
+    rocblas_int incy = options.incy;
+
+    //Number of elements in input vector X and input vector Y (default value is 5)
+    rocblas_int n = options.n;
+
+    if(n <= 0) //Edge condition check
+    {
+        std::cout << "Value of 'n' should be greater than 0" << std::endl;
+        return 0;
+    }
+
+    //Scalar value used for multiplication
+    float hAlpha = options.alpha;
+
+    //Adjusting the size of input vector X for value of stride (incx) not equal to 1
+    size_t sizeX = (n * incx) >= 0 ? n * incx : -(n * incx);
+
+    //Adjusting the size of input vector Y for value of stride (incy) not equal to 1
+    size_t sizeY = (n * incy) >= 0 ? n * incy : -(n * incy);
+
+    //Allocating memory for both the host input vectors X and Y
+    std::vector<float> hX(sizeX);
+    std::vector<float> hY(sizeY);
+
+    //Intialising random values to both the host vectors X and Y
+    helpers::fillVectorNormRand<float>(hX);
+    helpers::fillVectorNormRand<float>(hY);
+
+    std::cout << "Input Vectors (X)" << std::endl;
+    helpers::printVector(hX);
+
+    std::cout << "Input Vectors (Y)" << std::endl;
+    helpers::printVector(hY);
+
+    /*Initialising the values for vector hYGold, this vector will be used as a Gold Standard  
+    to compare our results from rocBLAS SAXPY funtion*/
+    std::vector<float> hYGold(hY);
+
+    //CPU function for SAXPY
+    for(int i = 0; i < n; i++)
+        hYGold[i * incy] = hAlpha * hX[i * incx] + hY[i * incy];
+
+    //Using rocblas API to create a handle
+    rocblas_handle handle;
+    rstatus = rocblas_create_handle(&handle);
+    CHECK_ROCBLAS_STATUS(rstatus);
+    {
+        //Allocating memory for both the both device vectors X and Y
+        helpers::DeviceVector<float> dX(sizeX);
+        helpers::DeviceVector<float> dY(sizeY);
+
+        //Tansfer data from host vector X to device vector X
+        herror = hipMemcpy(dX, hX.data(), sizeof(float) * sizeX, hipMemcpyHostToDevice);
+        CHECK_HIP_ERROR(herror);
+
+        //Tansfer data from host vector Y to device vector Y
+        herror = hipMemcpy(dY, hY.data(), sizeof(float) * sizeY, hipMemcpyHostToDevice);
+        CHECK_HIP_ERROR(herror);
+
+        //Enable passing alpha parameter from pointer to host memory
+        rstatus = rocblas_set_pointer_mode(handle, rocblas_pointer_mode_host);
+        CHECK_ROCBLAS_STATUS(rstatus);
+
+        //Saxpy calculation on device
+        rstatus = rocblas_saxpy(handle, n, &hAlpha, dX, incx, dY, incy);
+
+        CHECK_ROCBLAS_STATUS(rstatus);
+
+        /*Transfer the result from device vector Y to host vector Y, 
+        automatically blocked until results ready*/
+        herror = hipMemcpy(hY.data(), dY, sizeof(float) * sizeY, hipMemcpyDeviceToHost);
+
+        CHECK_HIP_ERROR(herror);
+    } // release device memory via helpers::DeviceVector destructors
+
+    CHECK_HIP_ERROR(herror);
+
+    //Release device memory via helpers::DeviceVector destructors

```

Comment:
I think these lines are duplicate and can be deleted

Reply:
Thanks for the comment Torre, I will make the necessary changes.